feat: add --import command-line option

OverMighty opened the pull request:
This pull request adds an `--import` command-line option (and an `-I` short option) which, when specified, makes the "Import from zip" dialog automatically open when the main window is shown, with the URL field's value set to the URL specified as command-line argument.

In order to add this feature, I had to **make `MainWindow::droppedURLs(QList<QUrl>)` public.** I don't know if you're okay with that or not.

Closes #2988

peterix commented:
To do that, you may have to change/customize the protocol in between.

These need to be updated:
https://github.com/MultiMC/MultiMC5/blob/develop/application/MultiMC.cpp#L274-L292
https://github.com/MultiMC/MultiMC5/blob/develop/application/MultiMC.cpp#L844-L863

Ideally, also fix the launch command to not be just a name of something. Make it `launch ID` instead of `ID` like it is now.

OverMighty commented:
Ok, I will work on this.

OverMighty commented:
Done. `activate` messages still tell the main process to show the main window, I replaced the `<id>` messages with `launch <id>`, and I added a new `import <url>` kind of message.

peterix commented:
Just one thing: I'll squash/rebase this to make it follow the commit message format MultiMC uses.

peterix commented:
Merged.
